### PR TITLE
Delete log forwarders API

### PR DESCRIPTION
**Pull request by r0mant**
It is not needed anymore b/c gravity uses Kubernetes to manage log forwarders via config maps.

I also replaced builtin router with httprouter which we use everywhere.

### CODE REVIEW

**Review by a-palchikov — APPROVED** [review]
lgtm

File: cmd/wstail/forwarders.go [hunk 1]
```diff
@@ -1,106 +1,107 @@
...
-			return nil
-		}
-		if info.IsDir() {
-			return filepath.SkipDir
-		}
-		log.Infof("removing %v", path)
-		return trace.Wrap(os.Remove(path), "failed to remove %v", path)
-	}); err != nil {
-		log.Warningf("failed to delete forwarder configuration files: %v", err)
+	configMap, err := client.ConfigMaps(systemNamespace).Get(
```
Comment (a-palchikov) [inline cmd/wstail/forwarders.go]:
```go
import "k8s.io/client-go/pkg/api"

client.ConfigMaps(api.NamespaceSystem).Get(...)
```

File: cmd/wstail/forwarders.go [hunk 2]
```diff
@@ -1,106 +1,107 @@
...
+		forwardersConfigMap, metav1.GetOptions{})
+	if err != nil {
+		return trace.Wrap(err)
```
Comment (a-palchikov) [inline cmd/wstail/forwarders.go]: (optional) Could use rigging to map the error

File: cmd/wstail/forwarders.go [hunk 3]
```diff
@@ -1,106 +1,107 @@
...
-		f, err := os.Create(path)
-		if err != nil {
-			return trace.Wrap(err)
-		}
-		_, err = f.WriteString(config(forwarder))
-		if errClose := f.Close(); errClose != nil {
-			log.Warningf("failed to close file %v: %v", path, errClose)
-		}
+	if len(configMap.Data) == 0 {
+		log.Infof("no log forwarders configured")
```
Comment (a-palchikov) [inline cmd/wstail/forwarders.go]: `Info`?

File: cmd/wstail/forwarders.go [hunk 4]
```diff
@@ -1,106 +1,107 @@
...
-const rsyslogInitScript = "/etc/init.d/rsyslog"
+	err := yaml.Unmarshal([]byte(data), &forwarder)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	err = ioutil.WriteFile(
+		forwarderFilename(forwarder),
+		forwarderConfig(forwarder),
+		0755)
```
Comment (a-palchikov) [inline cmd/wstail/forwarders.go]: `sharedExecutableMask`? Does it need to be executable? `sharedReadWriteMask` (`0666`)?

File: cmd/wstail/forwarders.go [hunk 5]
```diff
@@ -1,106 +1,107 @@
...
+	Metadata struct {
+		// Name is log forwarder name
+		Name string `json:"name" yaml:"name"`
+	} `json:"metadata" yaml:"metadata"`
+	// Spec defines log forwarder specification
+	Spec struct {
+		// Address is forwarding address
+		Address string `json:"address" yaml:"address"`
+		// Protocol is forwarding protocol
+		Protocol string `json:"protocol" yaml:"protocol"`
```
Comment (a-palchikov) [inline cmd/wstail/forwarders.go]: optional as in `omitempty`?

File: cmd/wstail/forwarders.go (hunk 2 above)
Comment (r0mant) [reply cmd/wstail/forwarders.go]: Yeah, I was considering it, but decided not to bring in another monstrous dependency just for a single line.

**Review by a-palchikov — APPROVED** [review]
lgtm

**Pull request merged**